### PR TITLE
Added reader db functionality

### PR DESCRIPTION
**Pull request by jeslin**
awkwardly added index.html to the root, make sure you look at that too, my bad!

### CODE REVIEW

**Comment by MinickLS** [comment]
Has anyone tested to see if this works before we merge? It looks good otherwise

**Comment by kaysabelle** [comment]
Yes, Jeslin tested this and I watched her test it.

File: radlibDemoApp/www/js/index.js [hunk 1]
```diff
@@ -69,14 +71,13 @@ function getParsed() {
 // testing
 function testing(){
```
Comment (kaysabelle) [inline radlibDemoApp/www/js/index.js:72]: Would we be able to move this into a different function with a different name? I know we have to keep this for future reference when we finish integration of adding readers into the UI. Other people's testing() functions are different...

Comment (kaysabelle) [inline radlibDemoApp/www/js/index.js:72]: We can leave this for now.

**Pull request merged**